client/lib/DomExtender.ts

```diff
@@ -95,14 +111,58 @@ const ExtensionFns = {
       nodePointer,
     );
   },
-  async $getComputedVisibility(): Promise<INodeVisibility> {
+  async $clearValue(): Promise<void> {
+    const { awaitedOptions } = awaitedPathState.getState(this);
+    const coreFrame = await awaitedOptions.coreFrame;
+    await Interactor.run(coreFrame, [
+      { click: this },
+      { keyDown: KeyboardKey.Meta, keyPress: KeyboardKey.a },
```

Comment:
Won't this only work on mac?

Reply:
I feel a little weird about adding code without tests and without other OS support into the codebase, but I get the notion. I don't know how to keep track of these so we sort them out later... unless we're ONLY doing this in domExtenders? I think this will make a bit more sense if we had the commands thing we talked about earlier